packages/handlebars/package.json

```diff
@@ -21,6 +21,9 @@
     "promised-handlebars": "^2.0.1"
   },
   "peerDependencies": {
-    "@frctl/fractal": "^1.1.0"
+    "@frctl/fractal": "^1.3.0"
```

Comment:
should not update the peerDependency version since we don't change any functionality

Reply:
Reverted